src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3094,64 +3090,62 @@ def lists_match(l1, l2):
-    ExpressRouteCircuit, ExpressRouteCircuitSku, ExpressRouteCircuitServiceProviderProperties, SubResource = \
-        cmd.get_models(
-            'ExpressRouteCircuit', 'ExpressRouteCircuitSku', 'ExpressRouteCircuitServiceProviderProperties',
-            'SubResource')
-    client = network_client_factory(cmd.cli_ctx).express_route_circuits
-    sku_name = '{}_{}'.format(sku_tier, sku_family)
-    circuit = ExpressRouteCircuit(
-        location=location, tags=tags,
-        service_provider_properties=ExpressRouteCircuitServiceProviderProperties(
-            service_provider_name=service_provider_name,
-            peering_location=peering_location,
-            bandwidth_in_mbps=bandwidth_in_mbps if not express_route_port else None),
-        sku=ExpressRouteCircuitSku(name=sku_name, tier=sku_tier, family=sku_family),
-        allow_global_reach=allow_global_reach,
-        bandwidth_in_gbps=(int(bandwidth_in_mbps) / 1000) if express_route_port else None
-    )
-    if cmd.supported_api_version(min_api='2010-07-01') and allow_classic_operations is not None:
-        circuit.allow_classic_operations = allow_classic_operations
-    if cmd.supported_api_version(min_api='2018-08-01') and express_route_port:
-        circuit.express_route_port = SubResource(id=express_route_port)
-        circuit.service_provider_properties = None
-    return sdk_no_wait(no_wait, client.begin_create_or_update, resource_group_name, circuit_name, circuit)
+class ExpressRouteCreate(_ExpressRouteCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        from azure.cli.core.aaz import AAZIntArg
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.bandwidth = AAZIntArg(
+            options=["--bandwidth"],
+            help="Bandwidth of the circuit. Usage: INT {Mbps,Gbps}. Defaults to Mbps.",
+            nullable=True,
+        )
+        return args_schema
 
+    def _cli_arguments_loader(self):
+        args = super()._cli_arguments_loader()
+        args = [(name, arg) for (name, arg) in args if name not in ("bandwidth_in_mbps", "bandwidth_of_circuit")]
+        return args
 
-def update_express_route(instance, cmd, bandwidth_in_mbps=None, peering_location=None,
-                         service_provider_name=None, sku_family=None, sku_tier=None, tags=None,
-                         allow_global_reach=None, express_route_port=None,
-                         allow_classic_operations=None):
+    def pre_operations(self):
+        args = self.ctx.args
+        args.sku_name = '{}_{}'.format(args.sku_tier, args.sku_family)
+        args.bandwidth_of_circuit = (args.bandwidth.to_serialized_data() / 1000.0) if args.express_route_port else None
+        args.bandwidth_in_mbps = (args.bandwidth) if not args.express_route_port else None
+        if args.express_route_port:
+            args.provider = None
+            args.peering_location = None
```

Comment:
Please add "if has_value(args.\<arg-name\>)" for each argument, it represents whether the user input the argument.